packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -252,7 +265,8 @@ function parseElementDirectives(
     ctx: ParserCtx,
     parsedAttr: ParsedAttribute,
     parent: ParentNode,
-    parse5ElmLocation: parse5.ElementLocation
+    parse5ElmLocation: parse5.ElementLocation,
```

Comment:
Since we're passing the entire `parse5.Element`, we can skip passing the `parse5.ElementLocation` here and pass the needed values to the parsers.  

Maybe something like this?

```
const { tagName, ElementLocation: { parse5ElmLocation } } = parse5Elm;

....

const node = parser(ctx, parsedAttr, parse5ElmLocation, prev, tagName);
```

Reply:
There is some processing to obtain `parse5ElmLocation` in `parseElement`

Reply:
Filed https://github.com/salesforce/lwc/issues/3108